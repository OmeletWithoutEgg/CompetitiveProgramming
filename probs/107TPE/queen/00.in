1046
-29150 -70949
26867 97418
66485 11900
31372 22017
-63965 72130
36729 -88179
74155 78258
-45943 -27358
42955 9705
-31241 32996
11928 -43752
6300 15906
-81052 -75771
52170 66386
-6628 -32310
-75650 -17396
69834 58869
27009 86672
-20192 -88596
-44094 17427
-61999 14601
-73908 -64818
47848 -84279
-73942 -93458
-32865 33334
-83726 -45196
83452 -65929
-12874 -84550
62668 -8036
-85986 14737
-65984 -34863
87433 -84420
20013 33280
15554 -76994
32301 57105
95084 53430
-83005 -23735
40071 62934
-41255 -64089
-76073 -84203
-8154 -18185
26559 11893
-48958 -55185
38391 -96477
-29392 -20601
91389 -99529
4948 1754
-26858 -87880
92751 -57452
11304 6817
-93968 -50972
67057 17940
-3898 53203
91360 10898
40076 -55829
-21275 53473
24598 -27548
-82198 -10726
-33193 17336
-29994 56132
379 42033
26136 76249
5844 -81955
-45860 -18153
-43383 -14755
-18428 -30935
-96962 60170
-80557 56969
-64300 4884
-88895 84014
-78391 -42142
58922 -70079
-68983 -9444
-70657 -41620
89579 42165
19839 65995
-49405 93601
55895 15876
-26807 -2458
-83408 -28989
14782 62400
74972 49046
-708 8349
63759 -10880
-7465 2362
29305 -47126
83766 43348
-85292 -77484
36422 15639
-24974 30466
28021 66392
21342 -48279
-24071 21249
-14275 11095
40150 12939
-27310 23380
4672 98121
-95124 3918
36870 -71460
81392 -10297
5565 -39926
-43157 -47793
-89358 76170
23438 -61742
32005 37112
32290 -37867
-86803 -62866
-63954 -23927
-93790 85557
-32242 -12687
-77128 -76072
30473 73428
-49974 -77766
-65840 -78481
-15454 -68148
94627 -73360
10240 87466
61803 43582
48657 8766
79985 74751
-5907 81429
57847 48538
45382 -52722
81294 -15450
1286 73421
97439 -36766
91525 470
-81336 42437
75358 68334
-79816 31785
-80409 -83311
54636 4834
-54244 -28567
-60626 69339
85904 -67211
-34470 -79288
-50112 -62458
-1486 95796
-10091 -34652
-31677 32342
-59714 -44936
-89540 56773
12823 -23974
-30311 4119
21935 -55642
-68173 18199
-80052 16811
-35818 20315
32548 15341
26012 -14739
36481 -13762
-92145 23573
13794 -2256
11097 54644
-60820 93440
58919 -523
-65337 -62330
-78297 65703
31535 -9510
-83023 79876
17816 -19061
-34934 26119
-67739 -1750
48 -35534
-36837 87200
71823 24857
17540 -7439
95187 -34159
-29263 -56540
-32024 53666
-54491 -33825
-9381 -52324
-27899 -50094
84976 10541
54286 42539
-23821 28668
53968 91297
-63133 -66844
-21350 31083
34638 -48373
-70039 92319
70589 -59540
33655 -73045
6816 98810
-54283 -74934
41104 -98703
-82228 52799
81558 65847
12652 54872
-22844 -23582
30197 -73291
-83431 36587
10134 -14287
-24710 82213
-43439 -20819
61187 3310
5631 8957
19505 -34511
-34594 -37657
-19596 -13386
56212 -90131
18243 -64935
33694 63455
1917 -2606
-92541 -83179
2290 98591
21347 77923
-30472 13672
80695 25968
-99360 -16352
-27869 -26694
-8910 -45505
-82173 97661
26219 -21510
83627 -5906
-56522 59108
68162 7298
-12666 -65239
36621 -52092
83342 97808
78003 -99222
52695 -62148
61719 71872
-16604 90996
10170 -44145
-40395 -22537
-81114 -64518
-29917 -99364
-72703 -46057
-76027 21765
-57145 -72667
-41459 612
-71190 -98446
-96352 51602
52428 -83487
-6986 -10154
-77597 -72840
32880 91854
86033 80013
-58769 11640
68186 -99503
-53916 -95725
18679 13108
-27713 10815
50685 -76695
-65366 -73919
77031 -64853
-19585 64564
51897 74635
-90811 8559
-55313 -93311
-13031 30049
-17177 73601
43815 12038
14570 -81077
48458 71231
53739 -74255
41062 -26921
-66800 5705
36920 -38307
96367 83189
17761 25393
-55783 98716
-29668 12434
-53057 28225
-57798 -94216
-66610 72091
-14108 -25403
53518 -34915
11229 53007
34785 -54613
-35601 -59069
-50999 -89655
-48586 91646
-40542 -65116
71097 72280
-81974 83640
70956 -20631
-33266 89897
-99962 -9166
75500 96244
62828 -59976
-27567 -48520
-26619 -15107
-83285 4814
-47592 -19618
-82441 -73378
46074 97244
87072 57383
40534 -59235
-87859 -787
63130 -97797
99767 -48715
4290 -27998
81868 -93770
7558 -68424
85948 78784
-57742 91097
-55851 1277
98408 -53789
-44074 46339
-51906 99989
43485 -28048
14814 89439
56078 -18021
-67123 -25822
-13349 58804
-87653 -45244
-46294 -1314
-27938 -93329
60552 -28525
78194 86941
-70367 76652
25587 -71370
7408 -89246
69624 47995
96386 9540
86354 -10026
-34852 -50788
67151 -95578
20945 74415
42271 -14272
-1425 18297
-4629 -56637
58835 51561
-13207 21239
-96588 42777
-76081 -64198
37740 42133
-69720 -63326
14555 -24015
38552 42389
-19594 93581
5303 65386
-48212 -87448
-72536 51894
96085 -29208
-78606 -70522
-24034 -76680
-42693 -2421
-42923 72620
-94962 -94093
60145 -57720
-8066 -96247
-54394 14907
-52931 92593
42316 -74535
-24745 39957
-30278 53984
-14732 -59491
-6600 -79559
-2600 -55947
14528 78431
32940 210
-93376 48830
-95515 83213
-18587 83851
-94502 -31684
-47921 60188
-43914 67869
-61039 -84951
-59775 22538
-84646 -76029
-71749 -34808
47663 63855
-3470 79610
-55663 -405
23610 -50842
-80321 -42263
17616 -91751
-46023 -77204
51947 -30983
50341 52611
-7850 15189
74637 32359
45593 99847
37041 60114
72836 -49396
20346 -24467
92750 50711
46886 50792
-24618 -57375
29018 54382
49648 78405
5525 99495
-30347 -77850
37228 7212
-2947 -28034
-46377 87453
62633 -37550
43648 37102
-75301 -94578
12235 72979
-16399 -74779
65060 13249
-42833 -84363
-67602 -16903
28866 11946
-17797 75143
-86762 -7750
-11137 41132
-80498 -12202
20524 31690
-76255 -39538
8618 -4493
74484 85876
55213 88026
86555 -82461
73109 -10723
-60546 52037
76895 -2213
85002 -89358
-9659 38963
-37591 74287
-51168 79165
-65877 -18851
77525 -10858
-31237 39140
-85105 -67507
63061 15638
42440 -59811
30742 22991
99094 84942
75856 -87108
-12688 -38795
-52286 53449
-97411 -9276
-10570 69039
79536 1536
-52090 -52505
-20017 -8964
-57224 9581
-18521 18301
38431 -59747
99574 38281
877 -24682
-75172 -21559
51671 -38952
63258 75082
63462 -26361
39801 64134
94369 56116
-5182 91781
20413 11817
-13074 2754
77145 32804
47604 7202
19843 -28089
4798 8316
41993 -78
-84663 40021
7008 71995
-10962 -42877
27981 97491
20222 -2035
-76812 88189
75679 44262
-44045 65023
70999 -55820
15749 -33594
-89868 69797
-84068 -46885
-36059 12890
60706 -80090
-79497 -13690
83641 -79546
1246 -16974
-8266 9108
71006 48110
-44342 -20918
20081 39710
26007 58257
-93058 19901
-71636 -31294
-75890 -95503
-51709 -32380
67686 1838
37704 -1365
73308 -41489
-96785 12089
-65662 69492
-34738 29038
77637 18768
7429 -19021
-79511 -32626
12647 -46076
50718 -1477
-27752 -28040
-33557 -44065
-427 44908
9525 -67744
-15493 39067
-37036 16416
71291 -71134
47745 -52065
11334 -26603
-17972 -52622
285 27727
72749 -35231
-13537 -82361
29676 -27093
23178 -9727
60855 41574
-31384 -22558
38837 17261
95038 -64191
22636 -54646
-38519 74769
-21133 75246
79126 -26514
19359 56101
-46941 -58293
-27693 73109
-6295 33231
-10924 -24262
85977 48770
64407 -32251
71278 -80729
-27036 40572
85296 -22410
-11487 89133
28813 24927
96213 -50737
45569 -28339
-96582 -16052
71979 59419
68109 23726
-52795 -79606
-8891 11405
64457 -16884
43561 -32763
-50584 7818
-91422 75333
-41524 -62974
-5 -78831
22713 38602
-26381 -53769
55746 91644
3581 64509
-6208 -70987
34967 33333
54136 36216
18935 -50913
-93472 31727
-12673 66599
69503 -62279
66057 -90821
27271 79581
-42385 64881
-69182 -64325
3075 50542
13081 24693
-86084 -42581
-31925 65474
41024 55659
75958 -19477
-1331 -89467
-66147 -72470
-30005 47297
-38089 18414
-68038 69715
-68742 43794
81094 -95623
-90910 -15447
-82889 -49558
46735 -49470
-54881 -83828
-32691 -21743
4611 94648
12299 41551
81043 23819
-3374 42896
-46741 39875
46393 -85215
2719 -95239
-84386 5325
68190 -82889
41382 53980
-99031 49864
38049 -72750
-44271 68747
-51277 59658
6430 84962
-21919 -46404
88202 -78426
21806 61781
-84062 -63793
-98595 79261
31333 -14888
76563 78411
2617 1390
79093 -75743
35908 -67464
-38593 50915
-89064 16138
91452 -11545
-25085 66071
-38130 -14796
-53230 90683
7563 -29204
-12163 -4829
-33533 -50341
67108 38920
-37234 36347
11920 -61864
-47447 -42369
86951 16087
35942 7415
19664 -75347
-82751 -32484
-42278 -6763
567 92317
41825 17943
9572 -59063
72576 97476
49339 83460
-95526 -409
-21436 35978
78028 68975
16349 -4277
90840 19553
20086 43812
-24666 -40239
47380 -73596
36588 -23647
86732 -81289
70842 13308
-23578 67755
-16203 -23237
-57955 80720
89278 55270
63985 -79492
20762 -16442
-63587 63111
60929 -84548
14605 -31281
-61628 -49068
-73186 -74503
-90911 24313
6473 -66092
-23799 -32607
-38695 -89076
-73470 753
78659 -68443
-69216 -81848
55310 -15857
25486 48295
-77308 9852
74604 62339
12945 -85082
33621 -73024
-47125 9077
-18375 51663
-45719 -13320
-56290 24554
-97257 -90195
-47266 15732
-41243 -9286
26474 44493
-27441 -85054
-29733 -58853
75250 75021
-57305 -57177
12976 66633
-99993 -86444
-21114 -87924
-42630 22625
-61946 -89590
-78111 -13882
81323 47615
11947 -22520
29422 43003
5437 49747
42956 26120
-70946 -95069
7015 30929
-19134 -60314
98075 -78255
-49980 -35603
31359 89934
-15113 20699
-61050 51639
88087 95040
-54060 92194
11107 -76079
-53032 -77828
-54445 -13149
67748 -81362
24352 -88368
20547 -24402
-88629 -61626
15994 -59649
-26313 71531
10305 44306
-89750 -91638
79889 -28172
-65335 30497
9403 83967
-95739 -48519
-87613 -82016
63927 -34948
24359 -4012
-2995 27063
64441 88868
-36969 3824
94956 96795
55459 -2739
-71296 -13344
20812 -50137
-72286 -95326
-45280 -55423
38126 -67290
-40891 -92187
45731 -91557
-82988 -99930
76013 -83691
-50355 34978
-84389 35914
-67733 43824
49604 -8601
-2018 33689
-8509 -49381
-40818 -14196
7960 45529
-91854 49595
-46691 -62405
-56890 -71599
52509 -77480
-25552 -5788
33973 -8630
-11808 -8468
-27609 2518
69415 -64967
-82264 -66155
32347 -59582
-40928 -39543
-37221 -38526
28585 70857
1898 19185
-92918 -8007
-5810 -17412
60307 -10991
-3644 -57167
-71467 73605
31932 65270
70921 52911
13669 20628
61713 -16726
41 -83478
-8537 89974
-55733 -16494
-62594 -83354
-99747 -45087
-13375 18105
76588 -43652
36721 -77394
-3370 -2775
-43533 -39742
-1626 99644
-8682 -90595
24144 -37578
48080 -95966
20840 -48025
-73888 75069
-54565 -23684
37581 68474
12173 -75516
99738 62032
42251 19195
-24953 73865
-98252 27143
36681 -47013
47776 -22257
59909 6202
47952 -80039
32089 -69378
27201 -84000
31421 -55297
-79038 85064
-51208 72455
-13168 -68572
-99449 99215
46408 33041
-41683 -78377
-18244 -233
-14034 -94672
83653 83103
-45556 49807
-25355 -5650
45409 2769
69847 68551
30242 -30904
-988 96045
4451 69114
69735 -3513
-6492 -3525
-69792 -38674
-69953 -14093
-37901 -71242
-68019 55997
47049 -29568
50311 25344
-43712 90569
-51316 -75049
6358 39138
-56260 -97584
-65005 -1260
63287 -7606
3983 10423
90039 -74602
-87576 56982
-78221 -17823
13883 -38731
-15009 -26362
28691 89059
-2105 15041
38206 88524
-55876 50487
-4915 78767
85287 72409
-5399 -92731
-15804 -18324
-79115 95378
-10841 72174
94125 36114
86889 41460
-43579 62266
-38171 56225
-73812 48939
-94435 -6865
97885 -23544
28554 -65521
-16146 -22055
67910 4957
22997 19101
72879 -39642
-70957 52351
-15346 -7838
-43807 11138
43917 84419
12211 44017
-76936 -78965
77030 43002
-58347 25824
-26434 -40432
-96076 -49244
35057 34960
76398 -25300
15513 90803
-53513 46112
-53129 -56456
-84429 81104
71661 -62052
-57788 95589
44170 -80866
-8591 -47424
-108 -87715
-77033 99238
60069 -86397
83269 -65865
-5531 19715
-53259 -2015
91505 98859
60559 -9157
-74799 9028
-49675 44240
-92927 11901
6331 -27105
50855 60914
7356 19331
-60792 2515
80304 -97505
97336 8262
-26344 67930
-72778 98477
-15663 17172
18941 -73560
-11306 65974
-3292 14902
88803 91146
51661 -67940
40134 -5284
18551 23379
58302 47190
90011 3929
31083 -40209
-9602 -34228
53703 -41228
29933 -86168
4996 -28903
-83047 48859
67384 49977
-64319 -38891
-96700 60502
44816 -79145
26379 -46588
97112 55578
-69108 25128
56199 -49972
56639 23239
-15309 26924
-19686 85303
90606 85943
88777 14349
-81689 29777
-66240 -47055
2264 68001
44509 -1653
37700 23746
52243 62374
-43811 43628
-79754 -96395
53402 45038
-28231 -40232
-83583 30788
-74508 -76665
61830 57859
22049 -62067
36093 18033
21300 51779
-61059 91801
-71744 38864
-56490 -27128
-66498 -27545
73756 57041
-69970 30129
-69856 -48144
53314 58907
-85440 74610
34086 78370
-41706 15002
59775 61351
-85807 -13616
-65431 75042
-60127 81933
96185 28959
99284 58235
36658 -51793
-34999 -53440
97607 19044
-15741 67736
-89509 71096
-3206 -88928
40838 77451
-58018 -31342
35255 26050
-58936 -52114
42416 -82926
-88921 44788
-68855 -41024
46399 29288
61793 4180
-87683 -95299
-87170 18394
-45994 36775
-8721 84632
79184 -19025
-56879 70348
76307 71596
-56113 22871
-59463 -70573
3894 -74891
27164 -10749
35227 24182
-21997 -79833
-23976 -75220
-54963 -75245
98163 3637
47699 68418
-51855 -59979
-61255 -64172
69196 -77378
24435 -74722
50956 -69514
22715 -33269
63008 -60190
20417 -21071
37460 17012
90641 34402
11404 -29409
-45847 52093
-81115 -63366
-26010 45041
-63162 16304
-24934 -13873
-71359 62290
60136 -52347
-64397 -85240
-37522 98443
30707 -46769
-82630 74340
56452 -83448
-1329 57376
65030 4386
-56512 90031
70168 -94406
3365 -37806
28505 -87191
-53641 -64781
88659 -55726
-28705 8419
-33274 36780
63495 16500
68564 91982
-79360 -18303
82645 -29669
78061 24366
74279 -36078
-41694 48135
3420 -62788
-69364 -97108
-83535 -44022
-65893 4812
-29405 10775
83544 -72126
51847 48259
22029 -61628
-47469 -13986
-39117 43587
-91501 82352
-43572 72796
-98839 75139
-46082 -29533
36796 -10432
13229 -69861
14774 -51283
-40816 65259
36884 -54620
-25573 -97531
-6519 -59938
110 86686
-2774 -80552
82838 61474
21829 21638
96131 54334
39263 -9574
-26042 -63518
-4366 -95902
-30254 -16152
90139 82131
40287 -98392
-28398 66884
-93311 -10335
-27085 64728
13023 23568
5654 -65160
