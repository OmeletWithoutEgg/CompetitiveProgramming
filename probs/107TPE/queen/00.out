-2008.78 32035.97
