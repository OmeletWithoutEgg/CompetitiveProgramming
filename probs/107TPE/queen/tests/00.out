-3434.25 -25004.77
